src/app/albums/albums.component.html

```diff
@@ -0,0 +1,13 @@
+<div class="container">
+    <div class="row">
+        <div class="col" *ngFor="let album of albums">
+            <div class="card" style="width: 18rem;">
+                <div class="card-body">
+                  <h5>ALBUM NAME: </h5>
+                  <h5 class="card-title">{{album.title}}</h5>
+                  <a [routerLink]="['/photos']" (click)="getPhotos(album.userId)" class="btn btn-primary">Show pictures</a>
```

Comment:
better solution would be to redirect to albums/:id/photos and get photos in that component